Google.PowerShell/BigQuery/GcbqDataset.cs

```diff
@@ -303,7 +303,7 @@ private class ParameterSetNames
         /// </para>
         /// </summary>
         [Parameter(Mandatory = false, ValueFromPipeline = true, ParameterSetName = ParameterSetNames.ByObject)]
-        public Dataset Dataset { get; set; }
+        public Dataset InputObject { get; set; }
```

Comment:
For consistency.  I did this because whenever i have an object that should be passed in via pipeline instead of by parameter, that's what I've been calling the variable.  I can go in and add aliases, but this is also the convention with other GCP cmdlets and it should be fine as is.